File: CSharp version/it/unical/mat/parsers/pddl/PDDLSolversParser.cs
using it.unical.mat.parsers.pddl.solver_planning_domains;

namespace it.unical.mat.parsers.pddl
{
    public static class PDDLSolversParser
    {
        public static string ParseSolverPlanningDomains(IPDDLDataCollection actions, string spdOutput, bool two_stageParsing)
        {
            return SPDGrammarBaseVisitorImplementation.Parse(actions, spdOutput, two_stageParsing);
        }
    }
}

File: Java version/app/src/main/java/it/unical/mat/parsers/pddl/PDDLSolversParser.java
package it.unical.mat.parsers.pddl;

import it.unical.mat.parsers.pddl.solver_planning_domains.SPDGrammarBaseVisitorImplementation;

public final class PDDLSolversParser {
	private PDDLSolversParser() {
		
	}
	
	public static String parseSolverPlanningDomains(final PDDLDataCollection actions, final String spdOutput, final boolean two_stageParsing) {
		return SPDGrammarBaseVisitorImplementation.parse(actions, spdOutput, two_stageParsing);
	}
}
